pkg/sdk/logging/model/output/opensearch.go

```diff
@@ -278,3 +294,34 @@ func (e *OpenSearchOutput) ToDirective(secretLoader secret.SecretLoader, id stri
 	}
 	return opensearch, nil
 }
+
+
+
+type OpenSearchEndpointCredentials struct {
+	// AWS region. It should be in form like us-east-1, us-west-2. Default nil, which means try to find from environment variable AWS_REGION.
+	Region string `json:"region,omitempty"`
+
+	// AWS connection url.
+	Url string `json:"url,omitempty"`
```

Comment:
nit: shouldn't this be mandatory?

Reply:
Looking at the plugin i think you are correct it should be. Fixing now.

Reply:
Changes pushed. Please take another look.